kubernetes/services/nginx-storage.yml.j2

```diff
@@ -11,7 +11,7 @@ spec:
         run: nginx-storage
     spec:
       containers:
-        - image:           nginx:1.13.6-alpine
+        - image:           {{ docker_registry }}/nginx-storage:latest
```

Comment:
I have merged @Gr33ny's branch that adds `image_prefix` and `image_version` today. Please add them here and rebase your branch on master.